src/error.rs

```diff
@@ -1,3 +1,28 @@
 use std::{error, result};
+use std::fmt::Display;
 
 pub type Result<T> = result::Result<T, Box<error::Error>>;
+
+pub trait OrExit {
```

Comment:
@0xd34d10cc did you invented this pattern or is it something common? :) Kinda like unwrap but with a custom message which is pretty convenient.